client2/src/shared/state/GUI.js

```diff
@@ -7,10 +7,12 @@ class GUI {
 
     cursorInTopSide = false;
 
+    chatInputStatus = false;
+
     constructor() {
         document.onmousemove = (event) => {
             this.setCursorX(event.clientX);
-            this.cursorY = event.clientY;
+            this.setCursorY(event.clientY);
```

Comment:
@Arcanorum Here's the fix to the bug that causes cursorY state to always return false